package.json

```diff
@@ -12,7 +12,8 @@
     "pre-version": "yarn test && yarn lint && yarn build",
     "release": "release-it",
     "serve": "yarn build:frame && vite",
-    "test": "jest"
+    "test": "jest",
+    "example": "vite"
```

Comment:
We already have a `vite` command two lines above that with the name of `serve`. You can use that to serve the example locally. It just needs to run `build:frame` before, otherwise you can't use anything.